Program.cs

```diff
@@ -294,6 +325,7 @@ private static void ReportSales()
             MenuMain();
             ProcessMain();
         }
+
         private static void ReportCustomersAge()
         {
             Console.WriteLine("Customers over 55");

```

Comment:
My idea with thes kind of rutine, is encapsule in a method which makes the question and return the value that the user wrote.